model.py

```diff
@@ -56,6 +56,7 @@ def from_name(cls, name: str):
     "30B": dict(n_layer=60, n_head=52, dim=6656),
     "34B": dict(n_layer=48, n_head=64, dim=8192, vocab_size=32000, n_local_heads=8, intermediate_size=22016, rope_base=1000000), # CodeLlama-34B-Python-hf
     "70B": dict(n_layer=80, n_head=64, dim=8192, n_local_heads=8, intermediate_size=28672),
+    "Mistral7B": dict(n_layer=32, n_head=32, n_local_heads=8, dim=4096, intermediate_size=14336, vocab_size=32002),
```

Comment:
Hi @alvion427 , thanks for your PR.
I tried and need to change `vocab_size=32000` to match the configs [here](https://huggingface.co/mistralai/Mistral-7B-v0.1/blob/main/config.json#L23).